mobify/progressive-web-scaffold
# Shipping form validation

This PR adds validation to the checkout shipping form. 

 **JIRA**: n/a
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/524

## Changes
- Add required field validation to shipping form
- Updated email and postal code Fields to use customEventHandler prop (fixes validation)

## How to test-drive this PR
- Run `npm install`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to the cart 
- proceed to the checkout shipping page
- confirm that form validation works as expected
